changes/66654a1a1a3aa7469ad4e64cfb0c9e5e.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: Something friendly about constant time tobool checks for container primtive
```

Comment:
`Improve performance of boolean evaluation of container types such as lists and dictionaries in Storm.`